pkg/agent/nodeportlocal/rules/netnat_rule.go

```diff
@@ -0,0 +1,116 @@
+//go:build windows
+// +build windows
+
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rules
+
+import (
+	"fmt"
+
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/agent/route"
+	"antrea.io/antrea/pkg/agent/util"
+)
+
+// Use antrea-nat netnatstaticmapping rules as NPL implementation
+const (
+	antreaNatNPL = "antrea-nat"
```

Comment:
Maybe we should define a global constant under /pkg/agent/util since this netnat is accessed by both Windows Route and NPL. Otherwise it is possible to introduce issues if any changes the name?

Reply:
Updated. Moved it to /pkg/agent/util/net.go